src/Router.tsx

```
@@ -19,61 +22,158 @@ import FriendPostPage from './pages/friend/FriendPostPage';
 import FriendDraftPage from './pages/friend/FriendDraftPage';
 import FriendSentTransitionPage from './pages/friend/FriendSentTransitionPage';
 
-import AnonDraftPage from './pages/letter/AnonDraftPage';
-import OtherDraftPage from './pages/letter/OtherDraftPage';
-import SelfDraftPage from './pages/letter/SelfDraftPage';
 import LetterDecoPage from './pages/letter/LetterDecoPage';
 import LetterReportPage from './pages/LetterReportPage';
-import LetterSendingPage from './pages/letter/LetterSendingPage';
 
+import OnboardingProfileSelectPage from './pages/onboarding/OnboardingProfileSelectPage';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'OnboardingProfileSelectPage'. Consider to delete it or rename it.

'OnboardingProfileSelectPage' is defined here:


(lint/suspicious/noRedeclare)